src/main/java/ch/njol/skript/conditions/CondIsTicking.java

```diff
@@ -0,0 +1,31 @@
+package ch.njol.skript.conditions;
+
+import ch.njol.skript.conditions.base.PropertyCondition;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import org.bukkit.entity.Entity;
+
+@Name("Is Ticking")
+@Description("Checks if an entity is ticking.")
+@Examples("send true if target is ticking")
+@Since("INSERT VERSION")
+public class CondIsTicking extends PropertyCondition<Entity> {
+
+	static {
+		register(CondIsTicking.class, "ticking", "entities");
+	}
+
+	@Override
+	public boolean check(Entity entity) {
+		return entity.isTicking();
```

Comment:
this is a PaperMC method, and will throw errors on Spigot (who uses spigot?)
Should add a check before registration.